README.md

```diff
@@ -1,33 +1,40 @@
-# Workshop: Monitoring Python Applications with Prometheus
+# Workshop: We know what your app did last summer. Do you? 👀
+## Observing Python Applications with Prometheus 🔥🐍  
 
-## Objective
+### Objective
 
-In the directory `app/`, we have a simple Python application. We want to start observing the behaviour of this application at runtime, by tracking and exporting metric data.
+In the directory `app/`, we have a application that runs a Python web server, with the endpoint `/treecounter`. It displays the total number of trees planted by Ecosia users. We want to start observing the behavior of this application at runtime, by tracking and exporting metric data.
 
 We will do this using the time-series database system [Prometheus](https://prometheus.io), which uses a "pull" method to extract data from running applications. This means that the applications need to "export" their data, so that Prometheus is able to "scrape" the metric data from them. This is typically done via an HTTP endpoint (`/metrics`, by convention).
 
 We will use the [Prometheus Python client library](https://github.com/prometheus/client_python) to track metrics in our code.
 
-## Agenda
+### Agenda
 
 * [Section 1: Exposing metrics](#section-1:-exposing-metrics)
 * [Section 2: Creating custom metrics](#section-2:-creating-custom-metrics)
 * [Section 3: Scraping Metrics with Prometheus and creating Dashboards with Grafana](#section-3:-scraping-metrics-with-prometheus-and-creating-dashboards-with-grafana)
 * [Bonus Material: Histograms in Prometheus](#bonus-material:-histograms-in-prometheus)
 
-## Prerequisites
+### Prerequisites
 
-For this workshop you will need [Python 3](https://installpython3.com/), [Pipenv](https://pipenv.pypa.io/en/latest/install/#installing-pipenv) and [Docker](https://docs.docker.com/get-docker/) running on your machine.
+For this workshop you will need [Python 3](https://installpython3.com/), [Poetry](https://python-poetry.org/docs/#installation), [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) running on your machine. *(on mac os docker-compose is by default installed with Docker)*
```

Comment:
Ah okay, good that I see this! I was going to add it as I was surprised it wasn't there yet. Makes sense <3